staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -60,6 +62,22 @@ func NewFieldManager(models openapiproto.Models, objectConverter runtime.ObjectC
 	}, nil
 }
 
+// NewCRDFieldManager creates a new FieldManager specifically for
+// CRDs. This doesn't use openapi models (and it doesn't support the
+// validation field right now).
```

Comment:
Are we keeping a list somewhere of things that need to be fixed?

Reply:
This list is mostly the spreadsheet that we used to have. As this gets released to alpha and we work on promoting to beta, I think it'd make sense to move some of the items that we have (and create the new items) in a github umbrella issue for transparency, etc.